internal/handlers/doctor/signup.go

```diff
@@ -85,6 +85,18 @@ func (d *doctorHandler) Signup(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+
+	// Sending notification to the doctor about registration
+	d.notificationRepository.Save(ctx,&models.Notification{
+		UserID: doctor.ID,
+		UserType: "doctor",
+		Title: "Welcome",
+		UserEmail: doctor.Email,
+		Message:"Thank you for Registering to ORDI!" ,
```

Comment:
We should send a similar notification to Patient as well.

Maybe you can have this message in a constant somewhere in util.

welcomeMessage = "Thank you for Registering to ORDI!"
and use it wherever needed.